includes/classes/ia.core.field.php

```diff
@@ -1316,6 +1316,8 @@ public function deleteFilesByFieldName($fieldName, $itemName, $files)
     public function deleteUploadedFile($fieldName, $itemName, $itemId, $fileName = null, $checkOwnership = false)
     {
         $fieldName = iaSanitize::paranoid($fieldName);
+        $itemName = iaSanitize::paranoid($itemName);
+        $itemId = iaSanitize::paranoid($itemId);
```

Comment:
I guess it's better to validate using (int) as it's ID field which is numeric type. Please recheck.